### DISCUSSION TITLE
Feature selection in Shapefile

### DISCUSSION THREAD

**Discussion by pks23** [pelicanmapping/osgearth#2662] [Support Q&A]
I am working on pick functionality for Shapefile features. I am able to select the drawables from the view, but not able to find the corresponding features from the layer. I have also enabled the indexing in the layer, but still not able to get the feature ids from the selected drawables. 

I would appreciate it if you could help me on this. Thanks in advance.

**Comment by gwaldron** [comment]
Can we see a code snippet? osgEarth version?
